public void testSimple() throws Exception { [EOL]     ObjectMapper mapper = new ObjectMapper(); [EOL]     String TEST = "test exception"; [EOL]     Map<String, Object> result = writeAndMap(mapper, new Exception(TEST)); [EOL]     Object ob = result.get("suppressed"); [EOL]     if (ob != null) { [EOL]         assertEquals(5, result.size()); [EOL]     } else { [EOL]         assertEquals(4, result.size()); [EOL]     } [EOL]     assertEquals(TEST, result.get("message")); [EOL]     assertNull(result.get("cause")); [EOL]     assertEquals(TEST, result.get("localizedMessage")); [EOL]     Object traces = result.get("stackTrace"); [EOL]     if (!(traces instanceof List<?>)) { [EOL]         fail("Expected a List for exception member 'stackTrace', got: " + traces); [EOL]     } [EOL] } <line_num>: 19,41
